i18n/en.toml

```diff
@@ -66,7 +66,7 @@ other = "Print entire section"
 [community]
 other = "community"
 [community_join]
-other = "Join the {{ .Site.Title }} community"
+other = "Join the {{ .Site.Title }} Community"
```

Comment:
My request was to use uppercase letter in German language only, not in English translation.
So i doubt whether this change + the change in the other language files makes sense.

Reply:
@h1z3y3 I am following the English translation as the basis of the Spanish translation, so if you change to:

```toml
other = "Join the {{ .Site.Title }} community"
``` 
then it also changes to:

```toml
other = "Únete a la comunidad {{ .Site.Title }}" 
```